### PR TITLE
base layout and file structure

### PR DESCRIPTION
**Pull request by jdbbaugh**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+node_module
```
Comment (colebryant) [inline .gitignore]: node_modules should be plural

Comment (alagrad94) [reply .gitignore]: We should also include .DS_store for the Mac users

**Review by alagrad94 — APPROVED** [review]

**Comment by THATBLADEBOY** [comment]
Sweet!

**Review by colebryant — APPROVED** [review]

**Review by THATBLADEBOY — COMMENTED** [review]
Good deal

**Pull request closed without merging**